ccm_web/client/src/components/UserEnrollmentForm.tsx

```diff
@@ -277,85 +277,83 @@ export default function UserEnrollmentForm (props: UserEnrollmentFormProps): JSX
     }
 
     return (
-      <Grid container className={classes.container}>
-        <Grid item xs={12} sm={9} md={9}>
-          {emailField}
+      <div className={classes.container}>
+        {emailField}
+        {
+          userExists !== undefined && (
+            <Paper className={`${classes.alert} ${classes.spacing}`} role='alert' variant='outlined'>
```

Comment:
No major changes here, I just removed the `Grid`s (since this is handled globally now), which pushed it down one level of indentation.